### PR TITLE
Updates to the sprint cycle and end of sprint release process

### PR DESCRIPTION
**Pull request by shefalijoshi**
### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? N/A
3. Command line build passes? Y
4. Changes have been smoke-tested? N/A
5. Testing instructions included? N/A

### CODE REVIEW

**Review by unlikelyzero — DISMISSED** [review]
### Reviewer Checklist
- [ ] Changes appear to address issue?
- [ ] Appropriate unit tests included?
- [ ] Code style and in-line documentation are appropriate?
- [ ] Commit messages meet standards?
- [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
- [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -296,19 +297,10 @@ these standards.
...
-* A short description of the issue encountered.
-* A longer-form description of the issue encountered. When possible, steps to
-  reproduce the issue.
-* When possible, a description of the impact of the issue. What use case does
-  it impede?
-* An assessment of the severity of the issue.
-
 Issue severity is categorized as follows (in ascending order):
 
 * _Trivial_: Minimal impact on the usefulness and functionality of the
```
Comment (akhenry) [inline CONTRIBUTING.md:302]: I'd like to align these definitions a little better with how we use them in our triage sessions:
* _Trivial_: Minimal impact on the usefulness and functionality of the software; a "nice-to-have." Visual impact without functional impact,
* _Medium_: Some impairment of use, but simple workarounds exist
* _Critical_: Significant loss of functionality or impairment of use. Display of telemetry data is not affected though.
* _Blocker_: Major functionality is impaired or lost, threatening mission success. Display of telemetry data is impaired or blocked by the bug, which could lead to loss of situational awareness.

**Review by akhenry — CHANGES_REQUESTED** [review]
@shefalijoshi  @unlikelyzero  This looks fantastic, really great. 

I've provided some clarity about the criticality levels we've been using during triage in an inline comment.

**Review by unlikelyzero — APPROVED** [review]

**Review by akhenry — APPROVED** [review]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A - documentation changes only
3. Code style and in-line documentation are appropriate? N/A - documentation changes only
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue)

**Pull request merged**